prov/verbs/configure.m4

```diff
@@ -44,6 +44,17 @@ AC_DEFUN([FI_VERBS_CONFIGURE],[
 				[verbs_rdmacm_happy=0])
 	      ])
 
+	AC_COMPILE_IFELSE([AC_LANG_PROGRAM(
+			   [
+			    #include <infiniband/verbs_exp.h>
+			   ],
+			   [
+			    return (IBV_EXP_DEVICE_ATTR_ODP | IBV_EXP_DEVICE_ODP);
```

Comment:
Ah, I assumed they were preprocessor macros.  But if they're enums, then your approach (just try to use them in a configure test) is probably the Right way to go.

In that case, you might want to always define OFI_HAVE_ODP_DEVICE_STUFF to 0 or 1 as the result of your configure tests.  Then you can always:

```c
#if OFI_HAVE_ODP_DEVICE_STUFF
/* ... etc. */
#endif
```

and never have the ambiguity of `#if defined(BLAH)` (e.g., if you accidentally have a typo in there, it'll never fail to compile -- it'll just always be false, which may be a difficult bug to find).